protoutil/blockutils.go

```diff
@@ -298,3 +299,39 @@ func searchConsenterIdentityByID(consenters []*cb.Consenter, identifier uint32)
 	}
 	return nil
 }
+
+func VerifyTransactionsAreWellFormed(block *cb.Block) error {
+	if block == nil || block.Data == nil || len(block.Data.Data) == 0 {
+		return fmt.Errorf("empty block")
```

Comment:
while it's generally good to have such check, I'm thinking that a nil block, or a block with nil data wouldn't ever reach this function. Are we able to actually exercise this line in any test?

Reply:
It does... see my comment above. 
https://github.com/hyperledger/fabric/pull/4490#discussion_r1373235343

Reply:
> while it's generally good to have such check, I'm thinking that a nil block, or a block with nil data wouldn't ever reach this function. Are we able to actually exercise this line in any test?

In a previous incarnation of this PR it would return nil. Yoav asked me to change to return error.

Reply:
ah very good - I didn't see the comment because the conversation was marked as resolved. Thanks!